### PR TITLE
added new PyYAML with intel 2015a

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: easybuild/easyconfigs/l/libyaml/libyaml-0.1.4-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Nils Christian <nils.christian@uni.lu>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'libyaml'
+version = '0.1.4'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libyaml/libyaml-0.1.4-intel-2015a.eb]: 0.1.5 is out, use that?

**Comment by boegel** [comment]
@JensTimmerman: looks fine, just use latest version of `libyaml`

File: easybuild/easyconfigs/l/libyaml/libyaml-0.1.4-intel-2015a.eb (hunk 1 above)
Comment (JensTimmerman) [inline easybuild/easyconfigs/l/libyaml/libyaml-0.1.4-intel-2015a.eb]: 0.1.6 is also out, so I should use that?

Comment (boegel) [inline easybuild/easyconfigs/l/libyaml/libyaml-0.1.4-intel-2015a.eb]: the homepage says "The current release of LibYAML: 0.1.5 (2014-02-04)."

I can see there's also a 1.0.6 tarball, but maybe this is not an official release? If you can find an announcement somewhere for 0.1.6, sure, use that.

Comment (JensTimmerman) [inline easybuild/easyconfigs/l/libyaml/libyaml-0.1.4-intel-2015a.eb]: http://pyyaml.org/ shows a release event

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9b27b363c43b42ae49bb for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JensTimmerman!

**Pull request merged**